plugins/inputs/postgresql_extensible/postgresql_extensible.go

```diff
@@ -18,11 +18,12 @@ import (
 
 type Postgresql struct {
 	postgresql.Service
-	Databases      []string
-	AdditionalTags []string
-	Timestamp      string
-	Query          query
-	Debug          bool
+	Databases          []string
+	AdditionalTags     []string
+	Timestamp          string
+	Query              query
+	Debug              bool
+	PreparedStatements bool `toml:"prepared-statements"`
```

Comment:
Please fix the TOML tag
```suggestion
	PreparedStatements bool `toml:"prepared_statements"`
```